### ISSUE TITLE
Glacier download flow issue

### ISSUE THREAD

**Issue by bitcoinhodler** [jazarija/icecube#6]
If this gets accepted into the formal Glacier Protocol (and I hope it does) then the bash script will itself be part of the Glacier download package. The user will need to manually download and verify the zip just like they do today before they can trust the bash script enough to run it.

Given that, it's not necessary for this script to re-download the zip file and re-verify the integrity. You can assume the zip file is already here and already verified.

**Comment by jazarija** [comment]
Good point. I guess it would make sense to change this if/when this is included into Glacier, since its crucial to know the location of the remaining glacier files (relative to icecube.sh) in order to install them properly.
